### PR TITLE
new site and copyright

### PR DESCRIPTION
**Pull request by MaksimChepelev**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -3,7 +3,7 @@
   "version": "0.1.3",
   "description": "",
   "author": "Zishan Iqbal",
-  "license": "Copyright (C) 2017 iotracks, inc. \nEclipse ioFog is provided under the Eclipse Public License (EPL) \nhttps://www.eclipse.org/legal/epl-v10.html",
+  "license": "Copyright (C) 2018 Edgeworx, Inc. \nEclipse ioFog is provided under the Eclipse Public License (EPL) \nhttps://www.eclipse.org/legal/epl-v10.html",
```
Comment (ekrylovich) [inline package.json]: Why Edgeworx????

Comment (MaksimChepelev) [reply package.json]: I got it from Kilton

**Comment by ilaryionava** [comment]
all references of iotracks & iofabric, fabric, etc. should be replaced

File: src/server/controllers/viewsController.js [hunk 2]
```diff
@@ -169,14 +169,14 @@ const activateAccountPage = function(req, res){
...
   		res.render("account/activateUserAccount", {
 			title: 'Activate account',
-			metaDescription : "Account activation page for iotracks software",
+			metaDescription : "Account activation page for iofog software",
 			metaCacheControl : "",
 			metaPragma : "",
 			metaKeywords : "",
-			metaAuthor : "iotracks, inc.",
-			metaCopyright : "2016 iotracks, inc.",
+			metaAuthor : "Edgeworks, Inc.",
```
[the remark this answers is not in the document]
Comment (MaksimChepelev) [reply src/server/controllers/viewsController.js]: Edgeworx, i asked Kilton

**Pull request merged**